notebooks/official/custom/custom_training_tensorboard_profiler.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<p>Create a specification for your custom training job, which consists of the following:</p><p> </p><ul><li><code>worker_pool_spec</code>&nbsp;: Specifies the machine type to use and whether to use a single or multiple machines.</li><li><code>python_package_spec</code>&nbsp;: Specifies the Python package that contains your custom training code. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Reply:
<p>Changed</p>